pyleoclim/utils/tsutils.py

```diff
@@ -76,7 +76,7 @@ def time_unit_to_datum_exp_dir(time_unit, time_name=None):
         exponent = 9
         direction = 'retrograde'
     else:
-        warnings.warn(f'Time unit {time_unit} not recognized. Defaulting to years CE')
+        warnings.warn(f'Time unit {time_unit} not recognized. Defaulting to years CE', stacklevel=4)
```

Comment:
this is so that the warning will point to the user's code, rather than to the line of code within pyleoclim which raises the warning

Reply:
Thanks. That is useful as this warning was cropping up constantly on my end due to my inadequate parsing of units (now fixed). I could make sense of it because I know the code, but this should help other users figure out what's happening on their end.